2019/CG-06.md

```diff
@@ -75,7 +82,20 @@ These are a few suggestions for accommodation including the distances to Igalia
     1. Find volunteers for note taking
     1. Adoption of the agenda
     1. Proposals and discussions
-       1. TBD
+       1. [Bulk Memory](https://github.com/WebAssembly/proposals/issues/18) (½ hr)
+         1. Passive element segment encoding
+       1. [Multi Value](https://github.com/WebAssembly/proposals/issues/19) (update) (¼-½ hr)
+         1. Poll: phase 4
+       1. [Threads](https://github.com/WebAssembly/proposals/issues/14) (1 hr)
+         1. [Add fence instruction](https://github.com/WebAssembly/tool-conventions/issues/59#issuecomment-490268705) (½ hr)
+       1. [Tail Calls](https://github.com/WebAssembly/proposals/issues/17) (¼ hr)
+       1. [Garbage Collection](https://github.com/WebAssembly/proposals/issues/16) (1 hr)
+         1. Split [typed imports](https://github.com/WebAssembly/gc/blob/master/proposals/gc/MVP.md#imports) and [exports](https://github.com/WebAssembly/gc/blob/master/proposals/gc/MVP.md#exports) out from GC proposal
+       1. Collaborative benchmark suite (½ hr)
+       1. [Exception handling](https://github.com/WebAssembly/proposals/issues/4) (½-1 hr)
```

Comment:
I think this better be 1 hour, and we'd like to talk about whether we should catch traps or not
cc @mstarzinger